pkg/api/kptfile/v1/validation.go

```diff
@@ -27,6 +27,13 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
+const (
+	// constants related to kustomize
+	kustomizationKind     = "Kustomization"
+	kustomizationAPIGroup = "kustomize.config.k8s.io"
+	kustomizationFile     = "kustomization.yaml"
```

Comment:
There are actually 3 valid kustomization file names - https://github.com/kubernetes-sigs/kustomize/blob/59c410a70af15ed330cfd5292b1a642692a7b773/api/konfig/general.go#L10-14

`kustomization.yaml` is the most common, but I wanted to point this out in case you wanted to exclude them all.

Reply:
Thanks @natasha41575 . I will try to use the constants from api pkg itself.